lib/Notifier/AdminNotifications.php

```diff
@@ -82,12 +70,11 @@ public function getName(): string {
 	 * @param INotification $notification
 	 * @param string $languageCode The code of the language that should be used to prepare the notification
 	 * @return INotification
-	 * @throws \InvalidArgumentException When the notification was not prepared by a notifier
-	 * @throws AlreadyProcessedException When the notification is not needed anymore and should be deleted
+	 * @throws UnknownNotificationException When the notification was not prepared by a notifier
 	 */
 	public function prepare(INotification $notification, string $languageCode): INotification {
 		if ($notification->getApp() !== 'admin_notifications' && $notification->getApp() !== 'admin_notification_talk') {
-			throw new \InvalidArgumentException('Unknown app');
+			throw new UnknownNotificationException('app');
```

Comment:
This one has no constructor documented, so here 'app' is the generic message for the Exception, is that intended?

Reply:
Didn't add any. Just using it for the "bad" unit tests this app has which want to test why it was unknown